src/main/java/az/edu/turing/service/impl/BookingServiceImpl.java

```diff
@@ -1,20 +1,102 @@
 package az.edu.turing.service.impl;
 
 import az.edu.turing.domain.dao.BookingDao;
+import az.edu.turing.domain.dao.FlightDao;
+import az.edu.turing.entities.BookingEntity;
+import az.edu.turing.entities.FlightEntity;
 import az.edu.turing.model.dto.BookingDto;
-import az.edu.turing.model.dto.request.CreateBookingRequest;
 import az.edu.turing.service.BookingService;
 
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
 public class BookingServiceImpl implements BookingService {
 
-    private final BookingDao bookingDao;
+    private BookingDao bookingDao;
+    private FlightDao flightDao;
 
-    public BookingServiceImpl(BookingDao bookingDao) {
+    public BookingServiceImpl() {
         this.bookingDao = bookingDao;
+        this.flightDao = flightDao;
+    }
+
+    @Override
+    public BookingDto createBooking(long flightId, String firstName, String lastName) {
+        Optional<FlightEntity> flight = flightDao.getById(flightId);
+        if (!flight.isPresent()) {
+            return null; // və ya xüsusi istisna
+        }
```

Comment:
not a good approach.
we can do this and handle it in controller or console:
`flightDao.getById(flightId)
.orElseThrow(() -> new NotFoundException("couldn't be able to find flight with id " + flightId));`